src/bugjira/issue.py

```diff
@@ -15,8 +17,14 @@ class Issue(BaseModel):
 
 
 class BugzillaIssue(Issue):
-    pass
+    @validator("key")
+    def validate_key(cls, key):
+        assert is_bugzilla_key(key)
```

Comment:
If this is only debugging code it is ok but if you expect this to run, avoid assert in code
https://docs.python.org/3/reference/simple_stmts.html#grammar-token-python-grammar-assert_stmt